libs/ngx-dsfr/header/src/lib/header.component.scss

```diff
@@ -5,6 +5,37 @@
 
 // Actual styling
 .fr-header {
+  .mini-mobile-header {
```

Comment:
The new version for this rule : 
```css
&.mini-mobile-header {
    .fr-header__brand {
      flex-wrap: nowrap;

      .fr-header__brand-top {
        width: auto;

        .fr-header__logo {
          padding-right: 0;
          margin-right: -8px;

          @media (min-width: 62em) {
            padding-right: 1rem;
            margin-right: 0.5rem;
          }

          .fr-logo {
            &::before {
              margin-bottom: -0.1rem;
              margin-right: 1rem;

              @media (min-width: 62em) {
                margin-bottom: 0.25rem;
                margin-right: unset;
              }
            }

            &::after {
              content: none;

              @media (min-width: 62em) {
                content: "";
              }
            }
          }
        }

        .institution,
        .fr-header__operator {
          display: none;

          @media (min-width: 62em) {
            display: block;
          }
        }
      }

      .fr-header__service {
        box-shadow: none;
        margin-left: 0;
        margin-right: 0;
        width: auto;
        padding: 1rem 0;

        @media (min-width: 62em) {
          padding: 1rem;
        }

        .fr-header__service-tagline {
          display: none;

          @media (min-width: 62em) {
            display: block;
          }
        }
      }
    }
```

It has the added benefit of deporting all the work on the CSS side and we therefore have less changes to make to the Template just for this feature